### DISCUSSION TITLE
Graph view: When the dag is large, the colored borders are hard to see

### DISCUSSION THREAD

**Discussion by MatrixManAtYrService** [apache/airflow#23225] [General]
### Apache Airflow version

2.3.0b1 (pre-release)

### What happened

I keep catching myself doing this:
1. zoom all the way out
2. look for the lime green borders
3. zoom in on that part of the dag

But on particularly large dags, I get stuck because the border is so thin that the color is easy to overlook.  For instance:

<img width="1773" alt="Screen Shot 2022-04-23 at 12 29 51 PM" src="https://user-images.githubusercontent.com/5834582/164936569-b191ca46-1451-47df-aa55-f10f4e10bdd3.png">

### What you think should happen instead

Can we get a thicker status-indicating border around these?

### How to reproduce

See above image

### Operating System

docker/debian

### Versions of Apache Airflow Providers

n/a

### Deployment

Astronomer

### Deployment details

n/a

### Anything else

no

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by bbovenzi** [comment]
I'm not sure if just making the border thicker really addresses this problem. The blue background of a group doesn't contrast enough with its children.
In a rebuild of the Graph view, I've been playing with the idea of removing zoom all together and replacing it with a minimap. The minimap could give a quick view of state or a button to center a complex DAG on the latest task instance.

**Comment by MatrixManAtYrService** [comment]
Now I'm imagining a "follow" mode, where the dag deforms to keep the currently running tasks in view, wherever they are.  Anyhow, it sounds like it's on your radar, and that's all I was really after with this issue.

**Comment by potiuk** [comment]
Ok. Let me convert it to discussion and we can always refer to it in the future.
